### PR TITLE
Provide option for Kinesis AWS region configuration

### PR DESCRIPTION
**Pull request by jtanza**
Addresses open [issue 97](https://github.com/openzipkin/zipkin-aws/issues/97). Currently, it is not possible to use a Kinesis stream created in any region outside of `us-east-1`. These changes allow users to set the AWS region for both Kinesis and STS independently, or optionally to the same region using a single environment variable, `AWS_REGION`.

### CODE REVIEW

File: autoconfigure/collector-sqs/README.md [hunk 1]
```diff
@@ -57,7 +57,6 @@ _Default 10_
 - `SQS_AWS_SECRET_ACCESS_KEY` Optional AWS Secret Access Key.
 - `SQS_AWS_STS_ROLE_ARN` Optional IAM role ARN for cross account role delegation.
 - `SQS_AWS_STS_REGION` Optional AWS region ID when using STS. _Default us-east-1_
-
```
[the remark this answers is not in the document]
Comment (jtanza) [reply autoconfigure/collector-sqs/README.md]: hrmf, how am i not seeing this locally?

```
jtanza @ ~/dev/zipkin-aws (master) $ git remote -v
origin	https://github.com/jtanza/zipkin-aws.git (fetch)
origin	https://github.com/jtanza/zipkin-aws.git (push)
upstream	https://github.com/openzipkin/zipkin-aws (fetch)
upstream	https://github.com/openzipkin/zipkin-aws (push)
jtanza @ ~/dev/zipkin-aws (master) $ git diff upstream/master autoconfigure/collector-sqs/README.md
jtanza @ ~/dev/zipkin-aws (master) $ 
```

File: collector-kinesis/src/main/java/zipkin2/collector/kinesis/KinesisCollector.java [hunk 2]
```diff
@@ -140,6 +151,38 @@ public void close() {
     worker.shutdown();
   }
 
+  public Collector getCollector() {
```
[the remark this answers is not in the document]
Comment (jtanza) [reply collector-kinesis/src/main/java/zipkin2/collector/kinesis/KinesisCollector.java]: yeap, exactly right, was being used in the autoconfiguration test. went ahead and changed it based on your suggestion.

**Comment by adriancole** [comment]
thanks a lot!

File: autoconfigure/collector-kinesis/src/main/resources/zipkin-server-kinesis.yml [hunk 3]
```diff
@@ -13,5 +13,9 @@ zipkin:
       aws-secret-access-key: ${KINESIS_AWS_SECRET_ACCESS_KEY:}
       # Optional AWS STS assume role ARN
       aws-sts-role-arn: ${KINESIS_AWS_STS_ROLE_ARN:}
-      # Optional AWS STS region endpoint to use
-      aws-sts-region: ${KINESIS_AWS_STS_REGION:us-east-1}
+      # Optional AWS STS region endpoint to use, defaults to KINESIS_AWS_REGION
+      aws-sts-region: ${KINESIS_AWS_STS_REGION:${zipkin.collector.kinesis.aws-kinesis-region}}
+      # Optional AWS Kinesis Region, defaults to AWS_REGION
+      aws-kinesis-region: ${KINESIS_AWS_REGION: ${zipkin.collector.kinesis.aws-region}}
```
Comment (zeagord) [inline autoconfigure/collector-kinesis/src/main/resources/zipkin-server-kinesis.yml]: The extra space in the nested yaml is broke. I will raise a PR with a couple of other changes soon. 
caused by: org.yaml.snakeyaml.scanner.ScannerException: mapping values are not allowed here
 in 'reader', line 19, column 47:
     ... sis-region: ${KINESIS_AWS_REGION: ${zipkin.collector.kinesis.aws ...

**Pull request merged**